src/ui/interaction/files/AvFileUpload/AvFileUpload.test.ts

```diff
@@ -31,11 +34,7 @@ describe('avFileUpload', () => {
 
     describe('when the component is mounted', () => {
       it('then it should render the slot content', () => {
-        expect(wrapper.text()).toContain('Upload a file')
-      })
-
-      it('then it should render the hint slot', () => {
-        expect(wrapper.text()).toContain('Accepted files: .pdf, .jpg')
+        expect(wrapper.text()).toContain('Ajouter un documentou glisser et deposer ici')
```

Comment:
```suggestion
        expect(wrapper.text()).toContain('Ajouter un document')
        expect(wrapper.text()).toContain('ou glisser et déposer ici')
```

correction de "déposer" et aussi pour bien montrer que c'est sur deux lignes et pas qu'il y a une erreur d'espace